packages/babel-jest/src/__tests__/getCacheKey.test.ts

```diff
@@ -9,6 +9,8 @@ import type {TransformOptions as BabelTransformOptions} from '@babel/core';
 import type {TransformOptions as JestTransformOptions} from '@jest/transform';
 import babelJest from '../index';
 
+const {getCacheKey} = babelJest.createTransformer();
```

Comment:
why not `babelJest.getCacheKey `? It should be on the default export, no?

Reply:
Thanks! What I'm alluding to here is that `babelJest.getCacheKey` should work fine, there's not need to call `createTransformer` first (I think!)

Reply:
And I am saying that functionality was removed in this PR 😄 See https://github.com/facebook/jest/pull/12407/files#diff-5f463bc5dbe428998302335614e9bbed3b4eb3310a6075767e1b432c161d9e71L273

It used to be:
```javascript

const transformer: SyncTransformer<TransformOptions> = {
  ...createTransformer(),
  createTransformer
}

export default transformer;
```

Note the ` ...createTransformer()`. That is what ensures that `babelJest.getCacheKey` exists. 

All of the fields from `...createTransformer()` would be ignored by Jest itself when it sees `createTransformer`. Since they are not used in any actual non-test code, I thought it did not make sense to expose them, since you could always just instantiate a new transformer in the tests. Once their use is gone from tests, no code would use them. So that's the reasoning behind removing them. I could keep them, of course, which would keep the tests unchanged, but what point is there in testing exports that will never be used?